README.md

```diff
@@ -465,17 +465,16 @@ Linux port of WinDirStat". Well, nothing could be further from the truth:
 **WinDirStat is a Windows port of KDirStat**, the predecessor of QDirStat.
 
 So it's the other way round: **The Linux version was there first**, and
-somebody liked it so much that he wrote a Windows version based on that
-idea. That's a rare thing; usually people port Windows originals to Linux.
+somebody liked it so much that they wrote a Windows version based on that
```

Comment:
I don't do that "they" for a single persion insanity.

Not only no, but hell, no!

Besides, I know for a fact that this somebody is a "he".